front/SettingsForm.tsx

```diff
@@ -90,6 +92,7 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               type="email"
               placeholder="Email"
+              data-cy="email-settings"
```

Comment:
Consider using a more descriptive name for the data-cy attribute, such as 'email-input'.